packages/terra-demographics-banner/docs/themeable-variables.md

```diff
@@ -0,0 +1,8 @@
+# Themeable Variables
+
+Variables to `theme` the terra-demographics-banner component.
+
+| Variable Name                       | Default value   | Usage                                                        |
+|-------------------------------------|-----------------|--------------------------------------------------------------|
+| $demographics-background            | $terra-blue-100 | Background of the Demographics Banner.                       |
+| $demographics-background-deceased   | #000            | Background of the Demographics Banner for a deceased person. |
```

Comment:
Please add this line too : https://github.com/cerner/terra-core/pull/191/files#diff-31058e8b65ca289134f485c9efa0ed3dR2